src/backend-api/Controllers/ListController.cs

```diff
@@ -234,7 +317,34 @@ public IActionResult GetListOfMonitors()
         [EnableQuery()]
         public IActionResult GetListOfPeripherals()
         {
-            return Ok(_context.Peripheral.ToList());
+            // TODO: This is a lot of repeated code from the last two endpoints. Maybe make the endpoints more generic??
+            // List that will be returned containing the list of peripherals
+            var listOfPeripherals = new List<object>();
+
+            // Employee list to be used later.
+            var Employees = _context.Employee;
```

Comment:
You don't have to do this I don't think as you can run lambdas and loops straight onto _context.employee